src/main/java/org/folio/circulation/domain/Location.java

```diff
@@ -0,0 +1,105 @@
+package org.folio.circulation.domain;
+
+import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import org.folio.circulation.support.JsonPropertyFetcher;
+
+import io.vertx.core.json.JsonObject;
+
+public class Location {
```

Comment:
Many of the methods in this class can be package-private please reduce the scope where possible

If the methods aren't easy to identify, please let me know and I can provide a list

Reply:
done